backend/status_codes.md

```diff
@@ -0,0 +1,53 @@
+# Status Codes
+
+Common cases
+--------------
+
+Status codes for common scenarios:
+
+* General:
```

Comment:
I think, `bad request` status is pending. It should have some like:
- `400 Bad request`:  The request could not be understood by the server due to malformed syntax (i.e Invalid data types).